src/components/molecules/TableComponent/TableComponent.tsx

```diff
@@ -15,55 +33,149 @@ const TableComponent: React.FunctionComponent<TableComponentPropsType> = ({
+          ? JAZZBAR_TABLES
+          : CONVERSATION_TABLES,
+    });
+
+    toggleModal();
+  }, [table.reference, venue.id, venue.template, toggleModal]);
+
+  const isDeleteButtonDisabled = isDeletingTable;
+
+  const isJazzBar = type === VenueTemplate.jazzbar;
+
+  const itemStyles = useCss(
+    isJazzBar
+      ? {}
+      : {
           height: `${table.rows && table.rows * 50 + 65}px`,
           width: `${table.columns && (table.columns + 1) * 55}px`,
-        }}
-      >
-        <div className="table-occupancy-information red-text">
+        }
+  );
+
+  const itemClasses = classNames("TableComponent__item", itemStyles);
+
+  const itemWrapperStyles = useCss({
+    width: isJazzBar ? "60%" : "65%",
+  });
+
+  const itemWrapperClasses = classNames(
+    "TableComponent__item-wrapper",
+    itemWrapperStyles
+  );
+
+  return (
+    <div className={`TableComponent ${table.reference}`}>
+      <div className={itemClasses}>
+        <div className="TableComponent__occupancy-warning">
           {locked ? "locked" : full ? "full" : ""}
         </div>
-        <div className="table-number">{table.title}</div>
+        <div className={itemWrapperClasses}>
+          {table.title}
 
-        {users &&
-          users.length >= 0 &&
-          users.map((user) => (
+          {isRemoveButtonShown && (
             <img
-              onClick={() => openUserProfileModal(user.id)}
-              key={user.id}
-              className="profile-icon table-participant-picture"
-              src={(!user.anonMode && user.pictureUrl) || DEFAULT_PROFILE_IMAGE}
-              title={(!user.anonMode && user.partyName) || DEFAULT_PARTY_NAME}
-              alt={`${
-                (!user.anonMode && user.partyName) || DEFAULT_PARTY_NAME
-              } profile`}
-              width={imageSize}
-              height={imageSize}
+              className="TableComponent__delete-icon"
+              src={PortalCloseIcon}
+              alt="remove table"
+              onClick={toggleModal}
             />
-          ))}
-        {users &&
-          table.capacity &&
-          table.capacity - users.length >= 0 &&
-          [...Array(table.capacity - users.length)].map((e, i) => (
-            <span
-              key={i}
-              onClick={() => onJoinClicked(table.reference, locked)}
-              id={`join-table-${venue?.name}-${table.reference}`}
-              className="add-participant-button"
-            >
-              +
-            </span>
-          ))}
+          )}
+        </div>
+        <div className="TableComponent__users">
+          {users &&
+            users.length >= 0 &&
+            users.map((user) => (
+              <img
+                onClick={() => openUserProfileModal(user.id)}
+                key={user.id}
+                className="TableComponent__profile-icon"
+                src={
+                  (!user.anonMode && user.pictureUrl) || DEFAULT_PROFILE_IMAGE
```

Comment:
There is a PR waiting for review that fixes issues with this 
- https://github.com/sparkletown/sparkle/pull/2727 

(especially since required to ignore the anon mode among other things)